### DISCUSSION TITLE
Pronunciation Adjustments

### DISCUSSION THREAD

**Discussion by melchiar** [DrewThomasson/ebook2audiobook#39] [Q&A] [answered]
Firstly, thank you for this fantastic project.

I'm wondering if there's a way to modify the pronunciation of specific words or names? (For instance, change the pronunciation of Gilbert from Gil-bair to Gil-burt)

**Comment by ROBERT-MCDOWELL** [comment, marked as answer]
you need to create your own fine tuned model for that

**Reply by melchiar** [reply]
Got it, thanks for the quick reply. 

So with something like this? https://github.com/daswer123/xtts-finetune-webui

**Reply by DrewThomasson** [reply]
## Yup 

- Thats a good repo 👍 

## you also

- could find a spelling where it'll say it how you want to correctly and do a replace-all 

- For instance replacing all instances of the word "dunce" with "DOONCE" lol

**Reply by melchiar** [reply]
nice, that's actually an excellent idea, thanks!
